--- ./usr/local/Aventail/startct.sh
+++ ./usr/local/Aventail/startct.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 XG_CLC_ARCHIVE=nui.jar
-xg_inst_dir=/usr/local/Aventail/nui
+xg_inst_dir=/opt/hwu-vpn/nui
 
 # If Java version predates this, we ask the user for confirmation before
 # running.
@@ -21,7 +21,7 @@
 }
 trap clean_up SIGINT SIGTERM
 
-XG_INSTDIR=/usr/local/Aventail
+XG_INSTDIR=/opt/hwu-vpn
 XG_MODULE=$XG_INSTDIR/modules/`uname -r`/xgdynroute.ko
 
 function warn_unsuitable_java {
--- ./usr/local/Aventail/startctui.sh
+++ ./usr/local/Aventail/startctui.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 XG_UI_ARCHIVE=nui.jar
-xg_inst_dir=/usr/local/Aventail/nui
+xg_inst_dir=/opt/hwu-vpn/nui
 
 # If Java version predates this, we ask the user for confirmation before
 # running.
@@ -25,7 +25,7 @@
 }
 trap clean_up SIGINT SIGTERM
 
-XG_INSTDIR=/usr/local/Aventail
+XG_INSTDIR=/opt/hwu-vpn
 XG_MODULE=$XG_INSTDIR/modules/`uname -r`/xgdynroute.ko
 
 function warn_unsuitable_java {
